exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/SelectTests.kt

```diff
@@ -564,4 +564,26 @@ class SelectTests : DatabaseTestsBase() {
             assertEquals(0, stringTable.selectAll().where { stringTable.name eq veryLongString }.count())
         }
     }
+
+    @Test
+    fun `test hint select query`() {
```

Comment:
Please use camelCase test naming and add the new test as a `@sample()` to the new `fun hint()`.

Please see [contribution style guide](https://jetbrains.github.io/Exposed/contributing.html#style-guides) for more details.